frontend/src/component/Header/MenuActionListItem/NotificationButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+require('./NotificationButton.styl')
+
+export const NotificationButton = props => {
+  return (
+    <div className='notificationButton'>
+      <button
+        className='notificationButton__btn btn outlineTextBtn nohover primaryColorBorder'
+        type='button'
+        onClick={props.onClickNotification}
+      >
+        <i className='fa fa-fw fa-bell-o' />
+        {props.t('Notifications')}
+        {props.notificationNotReadCount > 0 && (
+          <div
+            className='notificationButton__count'
+            style={{ fontSize: props.notificationNotReadCount > 99 ? '10px' : '11px' }}
```

Comment:
The size of the button and the bubble is always the same, the `10px` font-size is here because when 3 char need to be displayed `99+` in order to have enough space.
See: 
![11px](https://user-images.githubusercontent.com/31687635/89903626-2baee800-dbe8-11ea-9d32-4ecfb9f3bc5a.png)
and
![10px](https://user-images.githubusercontent.com/31687635/89903616-294c8e00-dbe8-11ea-8d17-a0474d8a6e1b.png)

Reply:
Yes, my question was if you tried 10px for the "2 char" numbers...

Reply:
Yes I tried, I thought that it was too small, but yes we should have a standard value after all
10px:
![10px](https://user-images.githubusercontent.com/31687635/89985559-8a717180-dc7b-11ea-9be5-c60389a26e60.png)

Reply:
Ok, thanks, if we think that it's too small when we use, we can change it easily that way.